bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -111,11 +115,15 @@
 		<description>A Plugwise Home Automation smart plug switch connected to a central heating pump</description>
 
 		<channels>
-			<channel id="power" typeId="power"/>
+			<channel id="power" typeId="system.power"/>
 			<channel id="lock" typeId="lock"/>
-			<channel id="powerUsage" typeId="powerUsage"/>
+			<channel id="powerUsage" typeId="system.electric-power"/>
```

Comment:
This label is particularly generic, "Electric Power".

Reply:
Yep, fixed.